openam-core/src/main/java/com/sun/identity/sm/DirectoryServerVendor.java

```diff
@@ -106,7 +109,10 @@ public Vendor query(Connection conn) throws LdapException, SearchResultReference
         Vendor vendor = unknownVendor;
 
         if (result != null) {
-            if (result.startsWith(VENDOR_OPENDJ)) {
+        	if (result.startsWith(VENDOR_WRENDS)) {
+    			String version = result.substring(VENDOR_WRENDS.length());
```

Comment:
Can you please replace tabs with spaces?

Reply:
Fixed🤦